### PR TITLE
HANS-293: Made /subscription endpoints part of private API

### PR DESCRIPTION
**Pull request by henrywrightPSC**

### CODE REVIEW

**Comment by henrywrightPSC** [comment]
https://github.com/nhsx/il-hans-infrastructure/pull/25

File: template.yaml [hunk 1]
```diff
@@ -27,18 +27,18 @@ Parameters:
   secretName:
     Type: String
     Default: "nhs-hans-staging"
+  vpcId:
+    Type: String
+  vpceId:
```
Comment (robhettrick-thepsc) [inline template.yaml:32]: Internal and External VPCs? I'd give a more verbose var name tbh. Single character difference with no description feels easy to miss.

**Review by robhettrick-thepsc — COMMENTED** [review]
@henrywrightPSC please check comment about VPCs

File: template.yaml (hunk 1 above)
Comment (robhettrick-thepsc) [reply template.yaml:32]: Just seen in the context of the other code that this is referring to the vpc endpoint, but it confusion around the var name is demonstrated I think...

**Review by robhettrick-thepsc — APPROVED** [review]

File: template.yaml (hunk 1 above)
Comment (henrywrightPSC) [reply template.yaml]: Yep - made more verbose!

**Pull request merged**